BUIP-UAHF/buip-uahf-technical-spec.md

```diff
@@ -151,6 +153,9 @@ NOTE 2: The client shall still accept transactions whose signatures
 verify according to pre-fork rules, subject to the additional OP_RETURN
 constraint introduced by REQ-6-1.
 
+NOTE 3: If bit 6 is not set, only the traditional non-shifted nHashType
```

Comment:
The wording came from the UAHF doc.  I originally took it from a jl2010 mailing list post.  What the code in btcfork actually does is mask off the top 3 bytes and set those 24 bits to the explicit forkid.

In a new tx we are actually setting, rather than adding or shifting, since the extra bytes come into existence when the 1-byte hashtype is widened to an int.